### PR TITLE
New metric schema

### PR DESCRIPTION
**Pull request by ami-GS**

### CODE REVIEW

**Comment by tiffanyfj** [comment]
You need to update your godeps.json and also the plugin version.

File: osv/cpu.go [hunk 1]
```diff
@@ -23,25 +23,26 @@ import (
...
 
 }
 
-func getCPUMetricTypes() ([]plugin.PluginMetricType, error) {
-	var mts []plugin.PluginMetricType
+func getCPUMetricTypes() ([]plugin.MetricType, error) {
+	var mts []plugin.MetricType
 	for _, metricType := range cpuMetrics {
-		mts = append(mts, plugin.PluginMetricType{Namespace_: []string{"osv", "cpu", metricType}})
+		mts = append(mts, plugin.MetricType{Namespace_: core.NewNamespace("osv", "cpu", metricType.Value)})
```
Comment (marcin-krolik) [inline osv/cpu.go]: Would be good to provide ```pluging.MetricType.Description_``` and ```pluging.MetricType.Units_``` whenever possible

**Comment by ami-GS** [comment]
Thanks!!

I'm not sure how to update Godeps.json because I have never used it.
I tried to manage godep and did unit test. Then I got errors as below.
`
gofmt
go vet
# testmain
runtime.raceinit: __tsan_init: not defined
runtime.raceinit: __tsan_map_shadow: not defined
runtime.racefini: __tsan_fini: not defined
runtime.racemapshadow: __tsan_map_shadow: not defined
runtime.racemalloc: __tsan_malloc: not defined
runtime.racegostart: __tsan_go_start: not defined
runtime.racegoend: __tsan_go_end: not defined
runtime.raceacquireg: __tsan_acquire: not defined
runtime.racereleaseg: __tsan_release: not defined
runtime.racereleasemergeg: __tsan_release_merge: not defined
runtime.racefingo: __tsan_finalizer_goroutine: not defined
runtime.RaceDisable: __tsan_go_ignore_sync_begin: not defined
runtime.RaceEnable: __tsan_go_ignore_sync_end: not defined
runtime.raceread: __tsan_read: not defined
runtime.racereadpc: __tsan_read_pc: not defined
runtime.racewrite: __tsan_write: not defined
runtime.racewritepc: __tsan_write_pc: not defined
runtime.racereadrange: __tsan_read_range: not defined
runtime.racereadrangepc1: __tsan_read_range: not defined
runtime.racewriterange: __tsan_write_range: not defined
runtime.racewriterangepc1: __tsan_write_range: not defined
/usr/lib/go/pkg/tool/linux_amd64/link: too many errors
FAIL    github.com/ami-GS/snap-plugin-collector-osv [build failed]
?       github.com/ami-GS/snap-plugin-collector-osv/osv [no test files]
`

**Comment by sandlbn** [comment]
@ami-GS thanks, closing this one after https://github.com/intelsdi-x/snap-plugin-collector-osv/pull/10 was merged.

**Pull request closed without merging**